File: ArtsofteMain/ClientApp/src/app/employees/employee.ts
import {Department} from "../departments/department";
import {Language} from "../languages/language";

export class Employee{
  constructor(
    public employeeID?: number,
    public name?: string,
    public surname?: string,
    public age?: number,
    public gender?: boolean,
    public department?: Department,
    public language?: Language
  ) {
  }
}

File: ArtsofteDAL/Models/Employee.cs
using System.Collections.ObjectModel;
using System.Diagnostics.CodeAnalysis;
 using System.Text.Json.Serialization;

 namespace ArtsofteDAL.Models
{
    [SuppressMessage("ReSharper", "InconsistentNaming")]
    public class Employee
    {
        public int EmployeeID { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public bool Gender { get; set; }
        public int Age { get; set; }
        public int DepartmentID { get; set; }
        public int LanguageID { get; set; }
        
        public Department Department { get; set; }
        
        public Language Language { get; set; }
    }
}